### DISCUSSION TITLE
Multi zone status

### DISCUSSION THREAD

**Discussion by razorclam** [gekkekoe/esphome-ecodan-hp#82] [General]
Hi,
Could someone please explain what the multi zone status is reporting?
My undertanding of my system is that it is set up on a single zone yet activity from Z2 and 'both' are reported.

![image](https://github.com/user-attachments/assets/3451b6ba-12b9-4c51-9cc5-86ff2dc06665)

**Comment by gekkekoe** [comment]
It's this byte https://github.com/gekkekoe/esphome-ecodan-hp/blob/main/protocol.md#0x03---refrigerant-and-zone-running-information. Are you sure your system is configured to have one zone only? Mines is reporting idle al the time since I only have 1 zone.

Did you include zone2.yaml in the build config ? And please report all the pump status.

**Reply by razorclam** [reply]
It's quite possible my understanding of the system is wrong. I am a learner after all.
I did not include Z2.
Below the pump values currently
![image](https://github.com/user-attachments/assets/dcfb6d1c-568e-436f-be0d-37bbf9601d1a)

**Reply by gekkekoe** [reply]
on the main ftc controller display, do you see z1 or z2 icons? (or [1] and [2] under the heating icon)

**Reply by razorclam** [reply]
Yes, it has this icon:
![image](https://github.com/user-attachments/assets/1edb32d9-e7a0-4949-82f0-cd0548b6aa4b)

**Reply by gekkekoe** [reply]
then its configured as 2 zones... I don't have those settings.

**Reply by gekkekoe** [reply]
enable zone2.yaml for other sensors and settings for 2 zone system

**Reply by razorclam** [reply]
Thanks. Was really easy to update. Thanks for your help
